controller_interface/src/controller_interface.cpp

```diff
@@ -27,24 +27,21 @@ namespace controller_interface
 return_type ControllerInterface::init(const std::string & controller_name)
 {
   node_ = std::make_shared<rclcpp::Node>(
-    controller_name, rclcpp::NodeOptions()
-                       .allow_undeclared_parameters(true)
-                       .automatically_declare_parameters_from_overrides(true));
-  lifecycle_state_ = rclcpp_lifecycle::State(
-    lifecycle_msgs::msg::State::PRIMARY_STATE_UNCONFIGURED, state_names::UNCONFIGURED);
-  return return_type::OK;
-}
+    controller_name, rclcpp::NodeOptions().allow_undeclared_parameters(true));
```

Comment:
Please do not chnage this.

```suggestion
    controller_name, rclcpp::NodeOptions()
                               .allow_undeclared_parameters(true)
                               .automatically_declare_parameters_from_overrides(true));
```